libsel4/sel4_arch_include/aarch64/sel4/sel4_arch/constants.h

```diff
@@ -273,10 +265,7 @@ SEL4_SIZE_SANITY(seL4_PUDEntryBits, seL4_PUDIndexBits, seL4_PUDBits);
 #if defined(CONFIG_ARM_PA_SIZE_BITS_44)
 #define seL4_UserTop 0x00000fffffffffff
 #elif defined(CONFIG_ARM_PA_SIZE_BITS_40)
-/* Currently other definitions of seL4_UserTop already have free slots at the end and don't need to subtract for seL4_VSpaceReservedSlots.
- * Each slot used requires subtracting 1GiB from the top address.
- */
-#define seL4_UserTop (0x000000ffffffffff - seL4_VSpaceReservedSlots * 0x40000000)
+#define seL4_UserTop 0x000000ffffffffff
```

Comment:
Could go for a more intuitive 
```suggestion
#define seL4_UserTop ((1 << 40) - 1)
```
now. Not sure of `(LIBSEL4_BIT(40) - 1)` also work, as this enforced 'ul' then.

Reply:
Using a plain old constant is what all other definitions use.

Reply:
Would agree that it makes more sense to stay consistent with the rest here, although I do generally appreciate the form that tells me where the constant is coming from. It's reasonably easy to see here, though, so not a big deal.